### DISCUSSION TITLE
Scheduler vs... scheduler

### DISCUSSION THREAD

**Discussion by k8gg** [sktaylortrash/TCPLightingWebInterface-MQTT-Docker#2] [Q&A]
I understand in the doc it reads: "_While components like the scheduler and IFTTT endpoints are technically still available in this project. They are not configured in this version and would require additional changes to the container._"

So this "scheduler" here... 
Does this mean we do not need to do the runSchedule.php every minute any more, per [https://github.com/sktaylortrash/TCPLightingWebInterface-MQTT/wiki/Setting-up-the-scheduler](url)...? 
... or does this mean the web interface scheduler (namely the scheduler.php) is not configured...?

**Comment by sktaylortrash** [comment]
It means that no method is available to trigger the running of schedules if you use the docker container. The scheduler is legacy code from the original project that I forked mine from and was never expected to be used in the MQTT-focused version. 
So because I have never used that portion, I did not want it running by default.  

To use the scheduler, you would need to make the changes to generate_sub.sh so it could run every 5 minutes or build a new script with a different wait time and add it to boot.sh.  Then you would need to rebuild the docker container yourself.  

I've updated the text on the manual install project to make it more clear why this new version exists. Also, you are welcome to continue using it as long as you have a host that can still run PHP 7.
